src/state/profile/index.tsx

```diff
@@ -17,7 +18,7 @@ export const profileSlice = createSlice({
     profileFetchStart: (state) => {
       state.isLoading = true
     },
-    profileFetchSucceeded: (state, action: PayloadAction<GetProfileResponse>) => {
+    profileFetchSucceeded: (_state, action: PayloadAction<GetProfileResponse>) => {
```

Comment:
what this change is about ? Please avoid making unneeded changes

Reply:
It was due to tsserver complaining about a declared but unused argument.
Due to your comment I looked into it, and noticed that VSCode doesn't make these warning obvious, but other editors such as Vim do, I hope this change is alright?

![image](https://user-images.githubusercontent.com/23725326/115025447-ab9cf880-9ec1-11eb-9fa4-ab550fc15d8c.png)

See [tsserver 6133](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#flag-unused-declarations-with---nounusedparameters-and---nounusedlocals).